src/install_pypi/mod.rs

```diff
@@ -203,18 +199,28 @@ impl<'a> PyPIPrefixUpdaterBuilder<'a> {
             &HashStrategy::None,
             &self.config_settings,
         );
+        let built_wheel_index = BuiltWheelIndex::new(
```

Comment:
We were not using this before this is to check for local wheel installs. "You: Were we really not using anything like that!?" "Me: ....yes"